operations/rollout-operator/rollout-operator.libsonnet

```diff
@@ -0,0 +1,371 @@
+
+    // Include the custom resource definiton for ReplicaTemplates.
+    replica_template_custom_resource_definition_enabled: $._config.rollout_operator_webhooks_enabled,
+
+    // Ignore_rollout_operator_*_webhook_failures will set the rollout-operator to ignore
+    // webhook failures. Useful during a rollout to a new cell, where rollout-operator service
+    // is still not created, as the webhook might be created before the service, and that could
+    // block other operations that would block the service creation.
+    ignore_rollout_operator_no_downscale_webhook_failures: false,
+    ignore_rollout_operator_prepare_downscale_webhook_failures: false,
+    ignore_rollout_operator_zpdb_eviction_webhook_failures: false,
+    ignore_rollout_operator_zpdb_validation_webhook_failures: false,
+  },
+
+  local enableWebhooks =
+    if $._config.rollout_operator_replica_template_access_enabled && !$._config.rollout_operator_webhooks_enabled then
+      error 'rollout_operator_replica_template_access_enabled requires rollout_operator_webhooks_enabled=true'
+    else if $._config.rollout_operator_replica_template_access_enabled && !$._config.rollout_operator_enabled then
+      error 'rollout_operator_replica_template_access_enabled requires rollout_operator_enabled=true'
+    else if $._config.rollout_operator_webhooks_enabled && !$._config.rollout_operator_enabled then
+      error 'rollout_operator_webhooks_enabled requires rollout_operator_enabled=true'
+    else if $._config.ignore_rollout_operator_no_downscale_webhook_failures && !$._config.rollout_operator_webhooks_enabled then
+      error 'ignore_rollout_operator_no_downscale_webhook_failures requires rollout_operator_webhooks_enabled=true'
+    else if $._config.ignore_rollout_operator_prepare_downscale_webhook_failures && !$._config.rollout_operator_webhooks_enabled then
+      error 'ignore_rollout_operator_prepare_downscale_webhook_failures requires rollout_operator_webhooks_enabled=true'
+    else if $._config.ignore_rollout_operator_zpdb_eviction_webhook_failures && !$._config.rollout_operator_webhooks_enabled then
+      error 'ignore_rollout_operator_zpdb_eviction_webhook_failures requires rollout_operator_webhooks_enabled=true'
+    else if $._config.ignore_rollout_operator_zpdb_validation_webhook_failures && !$._config.rollout_operator_webhooks_enabled then
+      error 'ignore_rollout_operator_zpdb_validation_webhook_failures requires rollout_operator_webhooks_enabled=true'
+    else if $._config.zpdb_custom_resource_definition_enabled && !$._config.rollout_operator_webhooks_enabled then
+      error 'zpdb_custom_resource_definition_enabled requires rollout_operator_webhooks_enabled=true'
+    else if $._config.replica_template_custom_resource_definition_enabled && !$._config.rollout_operator_webhooks_enabled then
+      error 'replica_template_custom_resource_definition_enabled requires rollout_operator_webhooks_enabled=true'
```

Comment:
Can these be rewritten as `assert` statements outside the context of `enableWebhooks`? It would help make the value of `enableWebhooks` clearer.